frontend/isoko/src/components/Review.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Rating } from 'react-simple-star-rating';
+
+const ReviewContainer = styled.div`
+   display: flex;
+   flex-direction: column;
+   align-items: start;
+   width: 40%;
+   padding: 0.5em;
+   margin: 0em 0em 1em 1em;
+   border-style: solid;
```

Comment:
Im fine with keeping borders, especially for now because we are still wokring on the rest of the business page. For the border tho, I would change it to 

border: 1px solid darkgrey; 
border-radius: 10px;